src/Peering/Peering/help/New-AzPeeringDirectConnectionObject.md

```diff
@@ -0,0 +1,222 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Peering.dll-Help.xml
+Module Name: Az.Peering
+online version: https://docs.microsoft.com/en-us/powershell/module/az.peering/new-azpeeringdirectconnectionobject
+schema: 2.0.0
+---
+
+# New-AzPeeringDirectConnectionObject
+
+## SYNOPSIS
+Creates a in memory PSObject to be used for creating or modifying a Peering.
+
+## SYNTAX
+
+### ParameterSetNameIPv4Prefix (Default)
+```
+New-AzPeeringDirectConnectionObject [-PeeringDBFacilityId] <Int32> -BandwidthInMbps <Int32>
```

Comment:
Please put the positional parameters in order of their position in the .cs file. This will ensure that when the syntax is produced in the help file that the positional parameters will show up in the right order, with any non-positionals afterward.